src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -536,7 +536,26 @@ public LocalWithInitializer(
                 Debug.Assert(initializer != null);
 
                 _initializer = initializer;
-                _initializerBinder = initializerBinder;
+                _initializerBinder = initializerBinder.GetBinder(initializer);
```

Comment:
>initializerBinder.GetBinder(initializer) [](http://example.com/codeflow?start=37&length=40)

It is expected to get null from this call for non-constant declarations. Most likely this is the reason why CI is failing.